packages/nifti-volume-loader/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "../../tsconfig.json",
```

Comment:
Didn't we find that referencing a global tsconfig file has issues with linking?

Reply:
no that was something else that I fixed in webpack